landoapi/api/landing_jobs.py

```diff
@@ -13,6 +13,36 @@
 logger = logging.getLogger(__name__)
 
 
+def get(landing_job_id: str):
+    """Return status about a landing job
+
+    Args:
+        landing_job_id (str): The unique ID of the LandingJob object.
+
+    Raises:
+        ProblemException: If a LandingJob object corresponding to the landing_job_id
+            is not found.
+    """
+    landing_job = LandingJob.query.get(landing_job_id)
+
+    if not landing_job:
+        raise ProblemException(
+            404,
+            "Landing job not found",
+            f"A landing job with ID {landing_job_id} was not found.",
+            type="https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/404",
+        )
+
+    if landing_job.status in (LandingJobStatus.LANDED,):
+        return {
+            "id": landing_job.id,
+            "status": landing_job.status.value,
+            "commit_id": landing_job.landed_commit_id,
+        }
+    else:
+        return {"id": landing_job.id, "status": landing_job.status.value}
```

Comment:
We could also disregard the status and return everything here, since `landing_job.landed_commit_id` will be an empty string in cases when there is a failure. This will keep the response more consistent.
```suggestion
    return {
        "id": landing_job.id,
        "status": landing_job.status.value,
        "commit_id": landing_job.landed_commit_id,
    }
```

Reply:
agreed